pcsc/src/lib.rs

```diff
@@ -571,6 +571,9 @@ impl Context {
         scope: Scope,
     ) -> Result<Context, Error> {
         unsafe {
+            #[cfg(target_os = "windows")]
+            let mut ctx: ffi::SCARDCONTEXT = null();
+            #[cfg(not(target_os = "windows"))]
             let mut ctx: ffi::SCARDCONTEXT = DUMMY_LONG;
```

Comment:
With my comment above, it will be possible to keep this line for windows, though might require `as ffi:SCARDCONTEXT` (not sure).